static/css/den.css

```diff
@@ -11,6 +11,8 @@
 
 @import url("syntax.css");
 
+@import url("friend.css");
```

Comment:
Instead of inserting this into the main css, you could insert it like this
https://github.com/shaform/hugo-theme-den/blob/f420b491a01699b8b97ee0a148f29fe14476209c/layouts/partials/head.html#L34

And add some checks to only include it when friend list shortcode is used: https://gohugo.io/templates/shortcode-templates/#checking-for-existence